cypress/support/getExcludedTags.js

```diff
@@ -0,0 +1,74 @@
+const getString = (v) => (typeof v === 'number' ? v.toString() : v)
+
+const extractMinorVersion = (v) =>
+    v.indexOf('2.') === 0 ? parseInt(v.slice(2, 4)) : parseInt(v.slice(0, 2))
```

Comment:
this looks like its made to support both `2.40` and just `40`. All examples of tags use the `xx` format, but dhis2 instances use the `2.xx` format. Should we maybe decide on which format to use to keep things consistent? or should the support for both count as a feature? 🙂

Reply:
I included both to offer some flexibility, but maybe it's better to just be strict about it from the get-go. I'm pretty sure the instance version will always be `2.xx` on CI. So we could probably simplify and enforce `2.xx`.